.gitignore

```diff
@@ -1,5 +1,6 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
+package-lock.json
+bun.lockb
```

Comment:
Sorry don't add bun.lockb here.
if you get conflicts on the file just delete it and run `bun i`